README.md

```diff
@@ -30,10 +30,14 @@ A Microverse exercise to use animation on a website
 - [📝 License](#license)
```

Comment:
### Optional
- [ ] It would be better if we customize our license file with our names and year